packages/flutter_dynamic_forms_components/lib/src/components/date/date_parser.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:dynamic_forms/dynamic_forms.dart';
+
+import 'date.dart';
+
+class DateParser extends ElementParser<Date> {
+  @override
+  String get name => "date";
+
+  @override
+  Date parse(
+      ParserNode parserNode, FormElement parent, ElementParserFunction parser) {
+    var date = Date();
+    date.fillDate(
+      id: parserNode.getPlainStringValue("id"),
+      isVisible: parserNode.getIsVisible(),
+      parent: parserNode.getParentValue(parent),
+      value: parserNode.getValue(
+          "value", (date) => DateTime.parse(date), () => null,
+          isImmutable: false),
+      format: parserNode.getStringValue("format", isImmutable: true),
+      initialDate: parserNode.getValue(
+        "initialDate",
+        (date) => DateTime.parse(date),
+        () => DateTime.now(),
+        isImmutable: true,
+      ),
+      firstDate: parserNode.getValue(
+          "firstDate",
+          (date) => DateTime.parse(date),
+          () => DateTime.parse("1969-07-20 20:18:04Z"),
```

Comment:
I think there is no need to call DateTime.parse and DateTime can be created directly, same applies to the lastDate

Reply:
Which one default value ? converter ? If you mean default value the time is int type so we need to parse it. ? am i wrong?

Reply:
Do you mean like this?
```
 firstDate: parserNode.getValue(
        "firstDate",
        (date) => DateTime.parse(date),
        () => DateTime(1969, 07, 20),
        isImmutable: true,
      ),
      lastDate: parserNode.getValue(
        "lastDate",
        (date) => DateTime.parse(date),
        () => DateTime(2050, 07, 20),
        isImmutable: true,
      ),
```

Reply:
yes, that's what I meant